pallets/eth-bridge/src/lib.rs

```diff
@@ -514,22 +523,30 @@ pub mod pallet {
         ) -> DispatchResultWithPostInfo {
             ensure_none(origin)?;
 
+            ensure!(events_fraction.is_valid(), Error::<T>::InvalidEventVote);
```

Comment:
do we want to specify that the events we got here are a "fraction" of the total? Would it be simpler if we called it `ethereum_events`?

Reply:
I think not. Given the definition of this variable:
```
events_fraction: DiscoveredEthEventsFraction,
```
Given its type, events_fraction is a clear name.